src/gasman.c

```diff
@@ -324,7 +324,7 @@ void CHANGED_BAG_IMPL(Bag bag) {
 #define SpaceBetweenPointers(a,b) (((UInt)((UInt)(a) - (UInt)(b)))/sizeof(Bag))
 
 #define SizeMptrsArea SpaceBetweenPointers(OldBags, MptrBags)
-#define SizeOldBagsArea SpaceBetweenPointers(YoungBags,OldBags)
+#define SizeOldBagsArea SpaceBetweenPointers(YoungBags, OldBags)
 #define SizeYoungBagsArea SpaceBetweenPointers(AllocBags, YoungBags)
 #define SizeAllocationArea SpaceBetweenPointers(StopBags, AllocBags)
 #define SizeUnavailableArea SpaceBetweenPointers(EndBags, StopBags)
```

Comment:
OK, looking at `StopBags` more closely was actually quite illuminating (thanks for stumping my nose on it, @stevelinton !). I discovered this line of code (around line 1958):
```
    /* temporarily store in 'StopBags' where this allocation takes us      */
    StopBags = AllocBags + WORDS_BAG(sizeof(BagHeader)+size);
```
So `StopBags` is indeed not what it "normally" is, and the `SpaceBetweenPointers` are misleading. I have therefore dropped those from this PR, and will instead look into removing StopGaps in another PR (I'll investigate it, anyway)